src/System.Windows.Forms.Design/src/System/ComponentModel/Design/Serialization/CodeDomComponentSerializationService.cs

```diff
@@ -1,16 +1,18 @@
 ﻿// Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
-using System.CodeDom;
 using System.Collections;
 
 namespace System.ComponentModel.Design.Serialization;
 
 /// <summary>
-///  This class serializes a set of components or serializable objects into a serialization store. The store can then
-///  be deserialized at a later date. CodeDomComponentSerializationService differs from other serialization schemes in
+///  On .NET Framework, this class serializes a set of components or serializable objects into a serialization store. The store can then
+///  be deserialized at a later time. The <see cref="CodeDomComponentSerializationService"/> differs from other serialization schemes in
 ///  that the serialization format is opaque, and it allows for partial serialization of objects. For example, you can
 ///  choose to serialize only selected properties for an object.
+///  On .NET, <see cref="CodeDomComponentSerializationService"/> can create a <see cref="SerializationStore"/> in memory,
+///  but that store can't be saved to a stream , not can a new store be loaded from a stream, as the
```

Comment:
Did you mean the following instead?
```suggestion
///  but that store can't be saved to a stream , nor can a new store be loaded from a stream, as the
```